thriftbp/server_middlewares.go

```diff
@@ -219,8 +219,15 @@ func ExtractDeadlineBudget(name string, next thrift.TProcessorFunction) thrift.T
 				v, err := strconv.ParseInt(s, 10, 64)
 				if err == nil && v >= 1 {
 					var cancel context.CancelFunc
-					ctx, cancel = context.WithTimeout(ctx, time.Millisecond*time.Duration(v))
+					timeout := time.Duration(v) * time.Millisecond
+					ctx, cancel = context.WithTimeout(ctx, timeout)
 					defer cancel()
+
+					client, _ := header(ctx, transport.HeaderUserAgent)
```

Comment:
Comment why it's ok to ignore the error?

Reply:
the dropped return is not an error, it's an `ok bool`